server/src/main/java/org/elasticsearch/lucene/spatial/GeoShapeCoordinateEncoder.java

```diff
@@ -11,7 +11,7 @@
 import org.apache.lucene.geo.GeoEncodingUtils;
 import org.elasticsearch.common.geo.GeoUtils;
 
-final class GeoShapeCoordinateEncoder implements CoordinateEncoder {
+public final class GeoShapeCoordinateEncoder implements CoordinateEncoder {
```

Comment:
same here: I would really like to keep it package private, why do you need it?

Reply:
These are used in ST_INTERSECTS when creating the geometry doc-values, as well as when creating the Component2DVisitor.

Reply:
You should use CoordinateEncoder.GEO

Reply:
OK. Done.